python/pyarrow/_compute.pyx

```diff
@@ -2251,3 +2325,158 @@ cdef CExpression _bind(Expression filter, Schema schema) except *:
 
     return GetResultValue(filter.unwrap().Bind(
         deref(pyarrow_unwrap_schema(schema).get())))
+
+
+cdef CFunctionDoc _make_function_doc(dict func_doc) except *:
+    """
+    Helper function to generate the FunctionDoc
+    """
```

Comment:
nit, but this docstring doesn't tell me anything that isn't already in the signature